lib/Badge.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Material badge for showing notifications, maybe more functionality in the future
+ * @flow
+ */
+
+import React, { Component } from 'react'
+import {
+  View,
+  Text,
+  StyleSheet
+} from 'react-native';
+
+type BProps = {
+  text: string,
+  size: number,
+  children: Array<ReactElement<*>>,
+  style: any,
+  isVisible: boolean,
+}
+
+const defaultProps = {
+  size: 20,
+  isVisible: true,
+  style: {},
+}
+
+export default class RippleBackgroundTransition extends Component {
+  static defaultProps: typeof defaultProps
+  props: BProps
+  static defaultProps = defaultProps
+
+  createStyles() {
+    const { size, style } = this.props;
+    const container = Object.assign(
+      {},
+      {
+        position: "relative",
+        width: size,
+        height: size,
+        borderRadius: size / 2,
+        alignItems: "center",
+        justifyContent: "center",
+        backgroundColor: "#F50057",
+        zIndex: 9999,
+        bottom: 5,
```

Comment:
_For android compatibility:_ Change `bottom` and `right` to something so it's correctly positioned. It worked for me with

```
top: 3,
left: '50%',
marginLeft: 15
```

Reply:
done